src/ONYXKEYS.ts

```diff
@@ -327,6 +327,7 @@ type OnyxValues = {
     [ONYXKEYS.WALLET_TERMS]: OnyxTypes.WalletTerms;
     [ONYXKEYS.BANK_ACCOUNT_LIST]: Record<string, OnyxTypes.BankAccount>;
     [ONYXKEYS.FUND_LIST]: Record<string, OnyxTypes.Card>;
+    [ONYXKEYS.CARD_LIST]: Record<string, OnyxTypes.Card>;
```

Comment:
We need different structure for both cards, no?
I mean OnyxTypes.Fund and OnyxTypes.Card

Reply:
I am pretty sure that's right, yes

Reply:
Also, why did we not have this line for `ONYXKEYS.CORPORATE_CARD_LIST`?

Reply:
> Also, why did we not have this line for ONYXKEYS.CORPORATE_CARD_LIST?

I'm not sure, it was added before the TS migration

Reply:
Pushed a fix